src/StructuredLogViewer.Avalonia/Controls/SearchAndResultsControl.xaml

```diff
@@ -33,7 +33,7 @@
         <ContentPresenter Name="watermark"
                    Margin="40"
                    HorizontalAlignment="Center"
-                   Content="{Binding WatermarkText}"
+                   Content="WatermarkText"

```

Comment:
I am not sure what is this supposed to bind to. It's necessary to provide a proper context for the compiled bindings.